python/pandemic_simulator/environment/infection_model/seir_infection_model.py

```diff
@@ -141,9 +141,9 @@ def __init__(self,
                  from_needs_hosp_to_death_rate: float = 0.3,
                  from_hosp_to_death_rate: Optional[float] = None,
                  spread_probability_params: Optional[SpreadProbabilityParams] = None,
-                 pandemic_start_limit: int = 5,
-                 numpy_rng: Optional[np.random.RandomState] = None):
-        self._numpy_rng = numpy_rng if numpy_rng is not None else np.random.RandomState()
+                 pandemic_start_limit: int = 5):
+        self._numpy_rng = globals.numpy_rng
+        assert self._numpy_rng, 'No numpy rng found. Either pass a rng or set the default repo wide rng.'
```

Comment:
yeah I got rid of the passing numpy_rng - because that was creating a confusion and doubts if the global one is used or one that's passed in. Entire repo-wide there is only one source of numpy rng, which can be reset by calling ps.init(seed=0)